activities/tool-management/nextcloud.md

```diff
@@ -24,6 +24,8 @@ Tweaks in the Nextcloud configuration can be made after SSHing into the machine
 
 ## Privacy and GDPR
 
-Nextcloud has [an extensive GDPR complaince kit](https://nextcloud.com/gdpr/).
+We run a self-hosted instance of Nextcloud on a server managed by the trusted [service provider Maadix in Spain.](https://maadix.net/en/terms-conditions)
```

Comment:
We implicitly trust all of our tools and providers, or else we wouldn't use them.

```suggestion
We run a self-hosted instance of Nextcloud on a server managed by our [service provider Maadix in Spain](https://maadix.net/en/terms-conditions).
```